package.json

```diff
@@ -30,7 +30,7 @@
     "e2e": "yarn workspace e2e-tests test",
     "format": "prettier '**/*.{js,ts,tsx,scss}' --write --loglevel warn",
     "prerelease": "yarn build",
-    "release": "electron-builder --mac",
+    "release": "electron-builder --publish never",
```

Comment:
In addition to dropping the `--mac` so it builds for the detected OS, the `--publish never` prevents it from seeking out a GitHub token env var to publish with and failing when it's not found. Our Actions Workflows generally handle the job for the GitHub-published ones.